docs/conf.py

```diff
@@ -89,7 +89,7 @@ def check_if_v046():
 
 # Multiversion settings
 
-smv_tag_whitelist = r'^v0\.4\.6|^v0\.5\.\d+$'
+smv_tag_whitelist = r'^v0\.4\.6|^v0\.5\.\d|^v0\.6\.\d+$'
```

Comment:
should this be a bit more general not to include every single release?

Reply:
I'm open to suggestions for improving it. The problem is that it must not match anything before `v0.4.6` since the docs were set up very different back then (if they even existed).

Reply:
The lazy in me says "ok then" :D

Reply:
I think that the problem will be easier in a few versions time - once `v0.10.0` is our oldest supported version, we can just check for two digits in the second character.